front/src/views/recovery-password/RecoveryPassword.html

```diff
@@ -0,0 +1,162 @@
+<v-card color="#F6F8FB" width="560px" class="mx-auto" elevation="0">
+  <v-row justify="center">
+    <v-card-title
+      primary-title
+      class="primary--text text--lighten-1"
+      color="primary lighten-1"
+    >
+      <h4>{{$t('recoverypassword.title')}}</h4>
+    </v-card-title>
+  </v-row>
+  <v-card-text>
+    <v-stepper non-linear alt-labels elevation="0" v-model="step">
+      <v-stepper-header>
+        <v-stepper-step :complete="step > 1" step="1">
+          <div class="text-center text-normal">
+            {{$t('recoverypassword.checkMail')}}
+          </div>
+        </v-stepper-step>
+        <v-divider></v-divider>
+        <v-stepper-step :complete="step > 2" step="2">
+          <div class="text-center text-normal">
+            {{$t('recoverypassword.title')}}
+          </div>
+        </v-stepper-step>
+        <v-divider></v-divider>
+        <v-stepper-step step="3">
+          <div class="text-center text-normal">
+            {{$t('recoverypassword.Conf')}}
+          </div>
+        </v-stepper-step>
+      </v-stepper-header>
+      <v-stepper-items>
+        <v-stepper-content step="1">
+          <v-card color="#F6F8FB">
+            <div class="text-center text-normal">
+              {{$t('recoverypassword.Entuemail')}}
+            </div>
+            <v-form ref="form" v-model="valid">
+              <div class="input-email">
+                <mmt-text-input
+                  class="input-field"
+                  mmt-label="E-mail"
+                  rounded
+                  filled
+                  single-line
+                  dense
+                  :placeholder="$t('recoverypassword.emailPlh')"
```

Comment:
emailPlh?? another part of this strange cosmic language?

Reply:
ninja